pkg/package-server-manager/controller.go

```diff
@@ -98,22 +99,31 @@ func reconcileCSV(log logr.Logger, image string, csv *olmv1alpha1.ClusterService
 	if csv.ObjectMeta.CreationTimestamp.IsZero() {
 		log.Info("attempting to create the packageserver csv")
 	}
-	if !validateCSV(log, csv) {
-		log.Info("updating invalid csv to use the default configuration")
-		tmp, err := manifests.NewPackageServerCSV(
-			manifests.WithName(csv.Name),
-			manifests.WithNamespace(csv.Namespace),
-			manifests.WithImage(image),
-		)
-		if err != nil {
-			return err
-		}
-		csv.Spec = tmp.Spec
+
+	expectedCSV, err := manifests.NewPackageServerCSV(
+		manifests.WithName(csv.Name),
+		manifests.WithNamespace(csv.Namespace),
+		manifests.WithImage(image),
+	)
+	if err != nil {
+		return err
 	}
-	if !ensureCSV(log, image, csv, highAvailabilityMode) {
+
+	if !ensureCSV(log, image, expectedCSV, highAvailabilityMode) {
 		log.V(3).Info("no further updates are necessary to the packageserver csv")
 	}
 
+	// Ensure defaults
```

Comment:
nit: Maybe expand a bit more on what it means to ensure these defaults here

Reply:
Done.